packages/run-protocol/src/econ-behaviors.js

```diff
@@ -241,7 +241,8 @@ export const startVaultFactory = async (
   {
     consume: {
       chainTimerService,
-      priceAuthority,
+      priceAuthority: priceAuthorityP,
+      reservePublicFacet: reservePublicFacetP,
```

Comment:
odd... usually we don't bother putting public facets in the produce / consume space, since we can get the instances via agoricNames and then the publicFacet via zoe

But if that's outside the scope of this PR...

to permit more stuff to be consumed, add it to https://github.com/Agoric/agoric-sdk/blob/master/packages/run-protocol/src/core-proposal.js#L40-L46

Reply:
thanks! [842b068](https://github.com/Agoric/agoric-sdk/pull/5343/commits/842b0687dc20107814831991d9ca76b6b59dfc64)